ziti/edge/api/client.go

```diff
@@ -754,6 +755,77 @@ func (c *ctrlClient) GetServices() ([]*edge.Service, error) {
 	return services, nil
 }
 
+func (c *ctrlClient) GetServiceTerminators(service *edge.Service) ([]*edge.Terminator, error) {
+	terminatorsUrl, err := url.Parse(fmt.Sprintf("/services/%v/terminators", service.Id))
+	if err != nil {
+		return nil, err
+	}
+	servReq, _ := http.NewRequest("GET", c.zitiUrl.ResolveReference(terminatorsUrl).String(), nil)
+
+	if c.apiSession.Token == "" {
+		return nil, errors.New("apiSession apiSession token is empty")
+	} else {
+		pfxlog.Logger().Debugf("using apiSession apiSession token %v", c.apiSession.Token)
+	}
+	servReq.Header.Set(constants.ZitiSession, c.apiSession.Token)
+	pgOffset := 0
+	pgLimit := 500
```

Comment:
nit: i couldn't find a const to reference in the sdk for this. should it be hardcoded like this? at least extract this to a const since it's in this file in 2 places now?

Reply:
Made paging explicit, so user can control it.